src/backend/bs_worker

```diff
@@ -3116,6 +3117,15 @@ sub dobuild {
       writestr("$srcdir/deltagen.spec", undef, $spec);
     }
   }
+  $bconf = $bconf || Build::read_config($buildinfo->{'arch'}, "$buildroot/.build.config");
+
+  my $useccache;
+  if (exists $bconf->{'buildflags:useccache'}) {
+      if (grep /^useccache:$packid$/s, @{$bconf->{'buildflags'} || []}) {
```

Comment:
Please use `/^useccache:\Q$packid\E$/` so that it also works if the packid contains a `+` sign